frontend/src/components/elements/DeckGlChart/DeckGlChart.tsx

```diff
@@ -46,14 +49,16 @@ interface PropsWithHeight extends Props {
 
 interface State {
   initialized: boolean
+  mapboxToken?: string
```

Comment:
why is it part of the state?

Reply:
It's fetched by DeckGLChart (as opposed to being passed in via props). We want to re-render the component when the token becomes available (or when there was an error fetching the token - this also lives in the state) - so it needs to be here.